plugins/ptp_operator/metrics/manager.go

```diff
@@ -207,7 +207,7 @@ func (p *PTPEventManager) PublishEvent(state ptp.SyncState, ptpOffset int64, eve
 		return
 	}
 	// /cluster/xyz/ptp/CLOCK_REALTIME this is not address the event is published to
-	eventSource = fmt.Sprintf(p.resourcePrefix, p.nodeName, eventSource)
+	eventSource = fmt.Sprintf(p.resourcePrefix, p.nodeName, fmt.Sprintf("/%s", eventSource))
```

Comment:
same comment, also looks better if you name function arg as `source` instead of `eventSource` to avoid assign an argument to itself.

Reply:
updated